src/main/java/mate/academy/internetshop/controller/GetAllOrdersController.java

```diff
@@ -13,7 +13,6 @@
 import mate.academy.internetshop.service.UserService;
 
 public class GetAllOrdersController extends HttpServlet {
```

Comment:
```suggestion
public class GetUserOrdersController extends HttpServlet {
```
такий неймінг буде правильніший

Reply:
Cool! Thanks!